app/server/controllers/UserController.js

```diff
@@ -538,6 +538,24 @@ UserController.sendPasswordResetEmail = function(email, callback){
     });
 };
 
+/**
+ * Acceptance into hackathon email
+ * @param  {[type]}   email    [description]
+ * @param  {Function} callback [description]
+ * @return {[type]}            [description]
+ */
+UserController.sendAcceptanceEmail = function(email, callback){
+  User
+    .findOneByEmail(email)
+    .exec(function(err, user){
+      if (err || !user){
+        return callback(err);
+      }
+
+      Mailer.sendAcceptanceEmail(email, callback);
```

Comment:
A parameter of this function is a callback, but you're not passing in a callback. Look above for more details.